bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitCommandExtension.java

```diff
@@ -119,18 +121,31 @@ private void listAccessories(Console console) {
         });
     }
 
+    private void printService(Console console, Service service, int indent) {
+        console.println(" ".repeat(indent) + "Service Type: " + service.getClass().getSimpleName() + " ("
+                + service.getType() + ")");
+        console.println(" ".repeat(indent + 2) + "Characteristics:");
+        service.getCharacteristics().forEach((c) -> {
+            try {
+                console.println(
+                        " ".repeat(indent + 4) + c.getClass().getSimpleName() + ": " + c.toJson(0).get().toString());
+            } catch (InterruptedException | ExecutionException e) {
+            }
+        });
+        if (service.getLinkedServices().isEmpty())
+            return;
```

Comment:
Please surround with {}

Reply:
is that a openhab style preference to always have braces with even single line conditionals? I've been doing that more, so if so I'll make sure to do it for future PRs.

Reply:
Yes, a good practice. 
I have never seen myself this way to code in OH code. 
I am not 100% sure but I believe it is detected by code style checks.